ch.threema.threema-desktop.desktop

```diff
@@ -0,0 +1,7 @@
+[Desktop Entry]
+Type=Application
+Name=Threema
+Icon=ch.threema.threema-desktop
+Exec=/app/bin/run.sh %U
```

Comment:
exec isn 't an absolute path

Reply:
Thanks for the fast review!

I admit, I simply adapted the .desktop from im.riot.Riot.  Will change that!